### ISSUE TITLE
Nemesis does not lower the cd of metamorphosis

### ISSUE THREAD

**Issue by Kim72** [mangosR2/mangos#653]
Nemesis is a demonology warlock talent.
When you spent 3/3 talentpoints the cooldown of demonic empowerment, fel domination and Metamorphosis should decrease with 30%.

The tooltip of Metamorphosis does display the 30% decrease and shows a cooldown of 2.1min.
And it is indeed possible to cast Metamorphosis after those 2.1min, HOWEVER you do NOY get the Metamorphosis buff and your meta is again on cd. But, when you cast Metamorphosis again not after 2.1min, but after 3min, it does work correctly.

The spell fel domination does work after the 2.1min cooldown and the spell demonic empowerment does work after the cd of 42sec.

In short: the talent Nemesis does not decrease the cooldown of metamorphosis.

Also reported here: http://getmangos.com/community/topic/16292/metamorphosis-is-not-working-as-it-should/
